judgments/tests/test_detail.py

```diff
@@ -601,3 +633,17 @@ def test_html_title_when_judgment(
         response = self.client.get("/eat/2023/1")
         html_title = "<title>Judgment A - Find case law</title>"
         assert_contains_html(response, html_title)
+
+
+class TestNoNeutralCitation(TestCase):
```

Comment:
I was about to say that this just covers line 89 in views/detail.py and line 106 is still untested.

However, we can actually just remove that whole block.

i.e remove  lines 103-106

```
    if context["document_noun"] == "press summary" and linked_document:
        if linked_document.best_human_identifier is None:
            raise NoNeutralCitationError(linked_document.uri)
        context["judgment_ncn"] = linked_document.best_human_identifier
```

since line 89 since we now use best_human_identifier